backend/src/main/java/org/eclipse/tractusx/puris/backend/delivery/domain/model/Delivery.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023 Volkswagen AG
+ * Copyright (c) 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V.
+ * (represented by Fraunhofer ISST)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.puris.backend.delivery.domain.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import org.eclipse.tractusx.puris.backend.common.domain.model.measurement.ItemUnitEnumeration;
+import org.eclipse.tractusx.puris.backend.common.util.PatternStore;
+import org.eclipse.tractusx.puris.backend.masterdata.domain.model.Material;
+import org.eclipse.tractusx.puris.backend.masterdata.domain.model.Partner;
+
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@SuperBuilder
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
+@Entity
+@ToString
+public abstract class Delivery {
+    @Id
+    @GeneratedValue
+    protected UUID uuid;
+
+    @ManyToOne()
+    @JoinColumn(name = "partner_uuid")
+    @ToString.Exclude
+    @NotNull
+    protected Partner partner;
+
+    @ManyToOne()
+    @JoinColumn(name = "material_ownMaterialNumber")
+    @ToString.Exclude
+    @NotNull
+    protected Material material;
+
+    private double quantity;
+    private ItemUnitEnumeration measurementUnit;
+
+    @Pattern(regexp = PatternStore.NON_EMPTY_NON_VERTICAL_WHITESPACE_STRING)
+    private String trackingNumber;
+
+    @Pattern(regexp = PatternStore.NON_EMPTY_NON_VERTICAL_WHITESPACE_STRING)
+    private String incoterm;
```

Comment:
Incoterm should be implemented as an Enum. The possible values can be found in the standard document.
![image](https://github.com/achtzig20/puris/assets/131770181/8dd06aba-9a62-46a0-bc5b-54d9c25b9120)